test_wazuh/test_fim/test_ignore/test_ignore_valid.py

```diff
@@ -89,20 +84,20 @@ def test_ignore_subdirectory(folder, filename, content, triggers_event,
     check_apply_test(tags_to_apply, get_configuration['tags'])
 
     # Create text files
-    create_file(REGULAR, folder, filename, content)
+    create_file(REGULAR, folder, filename, content=content)
 
     if get_configuration['metadata']['fim_mode'] == 'scheduled':
         # Go ahead in time to let syscheck perform a new scan
         TimeMachine.travel_to_future(timedelta(hours=13))
 
     if triggers_event:
-        event = wazuh_log_monitor.start(timeout=3,
+        event = wazuh_log_monitor.start(timeout=10,
```

Comment:
```suggestion
        event = wazuh_log_monitor.start(timeout=DEFAULT_TIMEOUT,
```

Reply:
In this case we are not using the DEFAULT_TIMEOUT value as we need a timeout of at least 10 seconds on Unix and the default one is 3.